phpunit.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not 100% sure if this is just a GitHub thing or not (I'm new to all of this!), but my local copy has an EOF new line after your review, but the diff doesn't display it. However, viewing the file states:

> 9 lines (8 sloc)  237 Bytes

So I guess it's worked after all.